tests/keras/backend/mxnet_sparse_test.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+from numpy.testing import assert_allclose
+import numpy as np
+import warnings
+import scipy.sparse as sparse
+
+from keras import backend as K
+
+
+BACKEND = None
+
+try:
+    from keras.backend import mxnet_backend as KMX
+    BACKEND = KMX
+except ImportError:
+    KMX = None
+    warnings.warn('Could not import the MXNet backend')
+
+
+class TestMXNetSparse(object):
+
+    @pytest.mark.skipif((K.backend() != 'mxnet'),
+                        reason='Testing only for MXNet backend')
+    def test_is_sparse(self):
+        x_d = np.array([0, 7, 2, 3], dtype=np.float32)
+        x_r = np.array([0, 2, 2, 3], dtype=np.int64)
+        x_c = np.array([4, 3, 2, 3], dtype=np.int64)
+
+        x_sparse_matrix = sparse.csr_matrix((x_d, (x_r, x_c)), shape=(4, 5))
```

Comment:
Lines 25 to 29 repeat across this file 8 times or so.
Doesn't it make sense to extract a common `generate_test_sparse_tensor` function instead or repeating the same code block?

Reply:
refactored